mailpoet/lib/Config/ServicesChecker.php

```diff
@@ -167,7 +167,7 @@ public function getAnyValidKey(): ?string {
   }
 
   public function generateHalfApiKey() {
-    $key = $this->getAnyValidKey();
+    $key = (string)($this->getAnyValidKey());
```

Comment:
I wonder if amending a previous commit instead of creating a new one for the fix would be better? What do you think?

Reply:
@pavel-mailpoet I believe i added this line in a new commit: `Fix phpstan error`

Reply:
I know. That is what I was trying to say. I wondered if it would be better to amend the commit that added this function instead of creating a new commit.

Reply:
@pavel-mailpoet Noted 👍

Reply:
@pavel-mailpoet  I'm not sure of a good way to make this change without destroying this branch and this PR. if this is not a blocker, I would like to leave it as it is and make a note of this for future reference.
Thanks

Reply:
Sure, leave it, not a big deal. But I think with an interactive rebase you could make the change. But if you are not familiar with it keep the commit you could lose your work and that is not worth it. I make such changes using git fixup but that is for the future changes: https://gist.github.com/JanJakes/b4e7fbc2df0c5d2db07b1971c1ecc403#simplifying-fixups

Reply:
Noted @pavel-mailpoet